src/lib.rs

```diff
@@ -701,10 +701,10 @@ impl Utf8Path {
     /// the current directory.
     ///
     /// * On Unix, a path is absolute if it starts with the root, so
-    /// `is_absolute` and [`has_root`] are equivalent.
+    ///   `is_absolute` and [`has_root`] are equivalent.
```

Comment:
Clippy complains if I don't comply to the newly added rule in 1.80.0:  https://github.com/camino-rs/camino/actions/runs/10400509135/job/28801185038?pr=93#step:4:150